lib/pages/Billing/Billing.tsx

```diff
@@ -30,8 +30,8 @@ const Billing: React.FC<BillingProps> = ({
   const trans = useTranslation()
   const [dialogOpen, setDialogOpen] = useState(false)
 
-  const t = (str: string) => {
-    return trans.t(`extensions.billing.${str}`)
+  const t = (str: string, otherParameters?) => {
+    return trans.t(`extensions.billing.${str}`, otherParameters)
```

Comment:
The way it is will not work as you expect. Spread the `args` instead. check [rest parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters) on MDN
```suggestion
    return trans.t(`extensions.billing.${str}`, ...args)
```